packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -156,13 +157,15 @@ async fn _run(
                             })?;
                         (Some(payload), Some(wrapper))
                     }
+                    InboundToRunnerMsgPayload::CancelTask(_) => {
+                        continue; // Don't send -- unused for now.
```

Comment:
Do we want to put a TODO, or an `unimplemented!()` here?

Reply:
I'll put a TODO. `CancelTask` is sent to the Python runner sometimes already, but it shouldn't be sent onwards to the Python process, because the message handling in Python hasn't been implemented (and no packages use it yet anyways).